spec/std/exception/call_stack_spec.cr

```diff
@@ -1,17 +1,5 @@
 require "../spec_helper"
```

Comment:
This file needs to require `./spec_helper`. I'm not sure if `../spec_helper` is necessary.

Reply:
I don't follow. That file does not exist.

Reply:
Sry, I mixed up the file location.